src/DIRAC/MonitoringSystem/Client/DataOperationSender.py

```diff
@@ -12,17 +12,20 @@
 from DIRAC.AccountingSystem.Client.Types.DataOperation import DataOperation
 from DIRAC.MonitoringSystem.Client.MonitoringReporter import MonitoringReporter
 
-sLog = gLogger.getSubLogger("__name__")
```

Comment:
Please leave it the way it is. There is no reason to create a instance logger everytime, and this way of doing is perfectly fine. Change the file back to what it was please

Reply:
Ok, but I suppose I should still remove the quotes around __name__, right ?

Reply:
definitely :laughing: